packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -326,17 +328,10 @@ function getPupilDistanceAndFaceHeight(face: Face) {
 
 export function generateBboxFromLandmarks(
   face: Face,
-  oval: LivenessOvalDetails
+  oval: LivenessOvalDetails,
+  frameHeight: number
```

Comment:
Pass in `frameHeight` which is used to limit `bottom` to be within the frame height